README.md

```diff
@@ -47,14 +47,13 @@ To learn more about the SDK, read:
 
 ## Sample projects
 
-For sample code, take a look at the following projects:
+For sample code, take a look at the following project:
 
-- [SampleApp (JavaScript)](https://github.com/Iterable/react-native-sdk/tree/master/SampleApp/javascript)
-- [SampleApp (TypeScript)](https://github.com/Iterable/react-native-sdk/tree/master/SampleApp/typescript)
+- [Example App](https://github.com/Iterable/react-native-sdk/tree/master/example)
```

Comment:
For the example app, is there a reason we only have one version of example app? I think there was a reason we had a typescript version and javascript version. We might need to think through providing one of each again.

Reply:
I spoke briefly to @Ayyanchira about this.  This is not a pattern I've ever seen in frontend development -- I'd venture to say that it actually makes it more difficult to understand.  I definitely would not use this pattern.

Reply:
Makes sense.